src/clientdb/SnmpGadgets.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 1996-2021 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+/* DEBUG: section 00    Client Database */
+
+#ifndef _SQUID__SRC_CLIENTDB_SNMPGADGETS_H
+#define _SQUID__SRC_CLIENTDB_SNMPGADGETS_H
```

Comment:
We (reluctantly) use "gadgets" when we cannot find a common word for the collection of assorted APIs. In this case, that word is "snmp", so there is no need to use saying-nothing "gadgets". Please rename to just `Snmp.{cc,h}` or, if you really want a second word here, `SnmpAccess.{cc,h}`.